src/deploy-cromwell-on-azure/scripts/mysql/init-user.sql

```diff
@@ -1,5 +1,5 @@
-CREATE USER 'cromwell'@'localhost' IDENTIFIED BY 'cromwell';
-GRANT ALL PRIVILEGES ON cromwell_db.* TO 'cromwell'@'localhost' WITH GRANT OPTION;
-CREATE USER 'cromwell'@'%' IDENTIFIED BY 'cromwell';
-GRANT ALL PRIVILEGES ON cromwell_db.* TO 'cromwell'@'%' WITH GRANT OPTION;
+CREATE USER '{MySqlUserLogin}'@'localhost' IDENTIFIED BY '{MySqlUserPassword}';
```

Comment:
Related to comment above, is there ever a time we want this to be replaced with empty string?

Reply:
No.